api/roleAccess/models/allocate-role-state-data.interface.ts

```diff
@@ -15,4 +15,5 @@ export interface AllocateRoleData {
   roleCategory?: RoleCategory;
   period: Period;
   jurisdiction?: string;
+  comment?: string;
```

Comment:
Regarding this field Tarik, can we make it a list of strings/objects as opposed to one string because there are multiple comments with regards to roles?